packages/app-store/stripepayment/lib/customer.ts

```diff
@@ -53,7 +53,7 @@ export async function getStripeCustomerId(user: UserType): Promise<string> {
       customerId = customer.id;
     }
 
-    await prisma.user.update({
+    await prisma.user.updateMany({
```

Comment:
As discussed, every profile with the same email will refer to the same `customerId` in Stripe, so we update all profiles to have that `customerId`